package.json

```
@@ -0,0 +1,33 @@
+{
+  "name": "crisis_web",
+  "private": true,
+  "version": "0.0.0",
```

Init with minor version
```suggestion
  "version": "0.1.0",
```